style.css

```diff
@@ -63,8 +63,12 @@
 
 .mobile-menu {
   background-color: #ebf0ee;
-  width: 95vw;
-  height: 75vh;
+  width: 100vw;
+  height: 100vh;
+  position: fixed;
+  left:0;
+  right:0;
+  top:0;
```

Comment:
- [ ] You have been presented with the original design in the previous to fix this pop-up window, you did some but not all of them, and they are still remarkable.
- the background color is not the correct one as in the original design
- the padding between the items is not the same as in the original design
- the font weight of the text is not the same as in the original design
- the hamburger is present but not well aligned as in the original design, there is some padding surrounding it.

<img width="804" alt="1" src="https://user-images.githubusercontent.com/77843050/209091040-8a0fee10-23f8-46b3-8312-863544da81fe.png">